src/pages/patientView/mutation/MutationInformationContainer.tsx

```diff
@@ -1,14 +1,66 @@
 import * as React from "react";
 import mockData from "./mock/mutationData.json";
+import {Td} from 'reactableMSK';
 import MutationTable from "../../../shared/components/mutationTable/MutationTable";
 import {IColumnDefMap} from "../../../shared/components/enhancedReactTable/IEnhancedReactTableProps";
+import {IColumnFormatterData} from "../../../shared/components/enhancedReactTable/IColumnFormatter";
+import TumorColumnFormatter from "./column/TumorColumnFormatter";
+import AlleleFreqColumnFormatter from "./column/AlleleFreqColumnFormatter";
 
 export interface IMutationInformationContainerProps {
     // setTab?: (activeTab:number) => void;
     store?: any;
+    sampleOrder:string[];
+    sampleColors:{ [s:string]: string};
+    sampleLabels:{ [s:string]: string};
+    sampleTumorType:{ [s:string]: string};
+    sampleCancerType:{ [s:string]: string};
```

Comment:
colors and labels are definitely things that should be passed down, they are defined in the context of the patient view. Tumor type and cancer type are more debatable, but I still think it's appropriate.